Jenkinsfile

```diff
@@ -1,3 +1,5 @@
+@Library('folio_jenkins_shared_libs@test-sonar-maven-update') _
```

Comment:
What is this?

Reply:
just some Sonar maven plugin testing/troubleshooting @OleksandrVidinieiev

Reply:
John is looking into the test issues that I asked about in the DM chat between the two of us Monday, @OleksandrVidinieiev